storybook/utils/storybookUtils.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * takes a function string and transforms it into a self invoking function string.
+ *
+ * @param {*} funcText
+ * @returns
+ */
+function makeSelfInvokingFuncString(funcText) {
+    return '(' + funcText + ')();'
+}
+
+/**
+ * gets rid of any special html characters and returns the intended user string.
+ *
+ * @param {string} html
+ * @returns
+ */
+function decodeHtml(html:string) {
```

Comment:
This isn't done automatically as part of displaying the story?

Reply:
not available in knobs plugin